.gitignore

```diff
@@ -4,3 +4,5 @@ docs
 coverage
 
 .DS_Store
+lsp/
```

Comment:
Where is this folder used?  Some kind of IDE?

Reply:
hi, 
it is Intelij LSP plugin https://github.com/gtache/intellij-lsp